data.sql

```diff
@@ -60,3 +60,223 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE  full_name = 'Bob') W
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE  full_name = 'Melody Pond') WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE  full_name = 'Dean Winchester') WHERE name IN ('Angemon', 'Boarmon');
+
+/*
+ Insert the following data for vets:
+ Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
+ Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
+ Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
+ Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
+ */
+ 
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+    ('William Tatcher', 45,'2000-04-23'),
+    ('Maisy Smith', 26, '2019-01-17'),
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-06-08');
+
+-- Insert the following data for specialties:
+-- Vet William Tatcher is specialized in Pokemon.
+INSERT INTO specializations (species_id, vet_id)
+SELECT species.id,vets.id
+FROM species,vets
+WHERE species.name = 'Pokemon'
+AND vets.name = 'William Tatcher';
+
+-- Vet Stephanie Mendez is specialized in Digimon and Pokemon.
+INSERT INTO specializations (species_id, vet_id)
+SELECT species.id,vets.id
+FROM species,vets
+WHERE species.name IN ('Digimon', 'Pokemon')
+AND vets.name = 'Stephanie Mendez';
+
+
+-- Vet Jack Harkness is specialized in Digimon.
+INSERT INTO specializations (species_id, vet_id)
+SELECT species.id,vets.id
+FROM species,vets
+WHERE species.name = 'Digimon'
+AND vets.name = 'Jack Harkness';
+
+
+/*
+Agumon visited William Tatcher on May 24th,
+2020.Agumon visited Stephanie Mendez on Jul 22th,
+2020.Gabumon visited Jack Harkness on Feb 2nd,
+2021.Pikachu visited Maisy Smith on Jan 5th,
+2020.Pikachu visited Maisy Smith on Mar 8th,
+2020.Pikachu visited Maisy Smith on May 14th,
+2020.Devimon visited Stephanie Mendez on May 4th,
+2021.Charmander visited Jack Harkness on Feb 24th,
+2021.Plantmon visited Maisy Smith on Dec 21st,
+2019.Plantmon visited William Tatcher on Aug 10th,
+2020.Plantmon visited Maisy Smith on Apr 7th,
+2021.Squirtle visited Stephanie Mendez on Sep 29th,
+2019.Angemon visited Jack Harkness on Oct 3rd,
+2020.Angemon visited Jack Harkness on Nov 4th,
+2020.Boarmon visited Maisy Smith on Jan 24th,
+2019.Boarmon visited Maisy Smith on May 15th,
+2019.Boarmon visited Maisy Smith on Feb 27th,
+2020.Boarmon visited Maisy Smith on Aug 3rd,
+2020.Blossom visited Stephanie Mendez on May 24th,
+2020.Blossom visited William Tatcher on Jan 11th,
+2021.
+*/
```

Comment:
- [Optional] Dear @Bornittah, It's good trying to comment but I see you overuse comments in your code base, This will make your code messy. Please remember comments are there to just explain the code not to overdo it. I highly recommend you remove/modify most of your comments.